src/seedu/addressbook/data/person/Address.java

```diff
@@ -50,10 +67,56 @@ public boolean equals(Object other) {
 
     @Override
     public int hashCode() {
-        return value.hashCode();
+        return this.value.hashCode();
     }
 
     public boolean isPrivate() {
         return isPrivate;
     }
 }
+
+class Block {
```

Comment:
Missing header comment. All non-trivial classes should have javadoc format header comments.